<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="left_verticalLayout">
        <item>
         <widget class="SourcePicturesWidget" name="files_tree" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="center_verticalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="DesktopWidget" name="desktop" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="desktop_controls_widget" native="true">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ElidedLabelWidget" name="mouse_pos_label" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ColorDisplayWidget" name="color_widget" native="true">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ElidedLabelWidget" name="color_value_label" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ElidedLabelWidget" name="desktop_status_label" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Scale: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="scale_doubleSpinBox">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>0.000000000000000</double>
             </property>
             <property name="maximum">
              <double>500.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="scale_horizontalSlider">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>5</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fit_image_to_desktop_pushButton">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="MainWindow.qrc">
               <normaloff>:/MainWindow/Resources/fit_to_size.ico</normaloff>:/MainWindow/Resources/fit_to_size.ico</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="right_verticalLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="tools_widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="handleWidth">
              <number>1</number>
             </property>
             <property name="childrenCollapsible">
              <bool>false</bool>
             </property>
             <widget class="ToolboxWidget" name="toolbox" native="true"/>
             <widget class="QTabWidget" name="tools_tabs_widget">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="tabPosition">
               <enum>QTabWidget::North</enum>
              </property>
              <property name="tabShape">
               <enum>QTabWidget::Rounded</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="documentMode">
               <bool>true</bool>
              </property>
              <widget class="LabelPropertiesWidget" name="label_editor">
               <attribute name="title">
                <string>Label</string>
               </attribute>
              </widget>
              <widget class="LabelDefinitionPropertiesWidget" name="definition_editor">
               <attribute name="title">
                <string>Definition</string>
               </attribute>
              </widget>
              <widget class="ProjectSettingsWidget" name="project_settings">
               <attribute name="title">
                <string>Project</string>
               </attribute>
              </widget>
              <widget class="ImagePropertiesWidget" name="image_properties">
               <attribute name="title">
                <string>Image</string>
               </attribute>
              </widget>
              <widget class="ImageSettingsWidget" name="image_settings">
               <attribute name="title">
                <string>Display</string>
               </attribute>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Projects">
     <property name="title">
      <string>Recent Projects</string>
     </property>
    </widget>
    <addaction name="new_project_action"/>
    <addaction name="separator"/>
    <addaction name="open_action"/>
    <addaction name="open_dataset_action"/>
    <addaction name="separator"/>
    <addaction name="save_action"/>
    <addaction name="save_as_action"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_Projects"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="fit_image_to_view_when_selected_action"/>
    <addaction name="enable_toolbox_filter_action"/>
    <addaction name="separator"/>
    <addaction name="action3d_View"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">QToolBar {
	border: none;
}
</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_action"/>
   <addaction name="open_dataset_action"/>
   <addaction name="save_action"/>
   <addaction name="separator"/>
   <addaction name="undo_action"/>
   <addaction name="redo_action"/>
   <addaction name="separator"/>
   <addaction name="evaluate_in_roi_action"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="save_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/save.ico</normaloff>:/MainWindow/Resources/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="open_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/open.ico</normaloff>:/MainWindow/Resources/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open Local Project...</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionRecent_Projects">
   <property name="text">
    <string>Recent Projects</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="fit_image_to_view_when_selected_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit image to view when selected</string>
   </property>
  </action>
  <action name="action3d_View">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/3d.ico</normaloff>:/MainWindow/Resources/3d.ico</iconset>
   </property>
   <property name="text">
    <string>Open 3d View Window</string>
   </property>
  </action>
  <action name="undo_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Undo.ico</normaloff>:/MainWindow/Resources/Undo.ico</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="redo_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Redo.ico</normaloff>:/MainWindow/Resources/Redo.ico</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="open_dataset_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/folder_network.ico</normaloff>:/MainWindow/Resources/folder_network.ico</iconset>
   </property>
   <property name="text">
    <string>Open DataSet Project...</string>
   </property>
  </action>
  <action name="enable_toolbox_filter_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable toolbox filter</string>
   </property>
  </action>
  <action name="project_settings_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/configuration.ico</normaloff>:/MainWindow/Resources/configuration.ico</iconset>
   </property>
   <property name="text">
    <string>Project settings</string>
   </property>
  </action>
  <action name="new_project_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/new.ico</normaloff>:/MainWindow/Resources/new.ico</iconset>
   </property>
   <property name="text">
    <string>New Project...</string>
   </property>
  </action>
  <action name="save_as_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/save.ico</normaloff>:/MainWindow/Resources/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save Project As...</string>
   </property>
   <property name="toolTip">
    <string>Save Project As</string>
   </property>
  </action>
  <action name="evaluate_in_roi_action">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/search-inside.ico</normaloff>:/MainWindow/Resources/search-inside.ico</iconset>
   </property>
   <property name="text">
    <string>Evaluate in ROI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorDisplayWidget</class>
   <extends>QWidget</extends>
   <header location="global">ColorDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SourcePicturesWidget</class>
   <extends>QWidget</extends>
   <header>SourcePicturesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolboxWidget</class>
   <extends>QWidget</extends>
   <header>ToolboxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>LabelPropertiesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelDefinitionPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>LabelDefinitionPropertiesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProjectSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ProjectSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ImageSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DesktopWidget</class>
   <extends>QWidget</extends>
   <header>DesktopWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElidedLabelWidget</class>
   <extends>QWidget</extends>
   <header>ElidedLabelWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImagePropertiesWidget</class>
   <extends>QWidget</extends>
   <header>ImagePropertiesWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
